appveyor.yml

```diff
@@ -1,7 +1,7 @@
 version: 1.0.{build}
 
 environment:
-    MXNET_VS: "Visual Studio 12 2013 Win64"
+    MXNET_VS: "Visual Studio 14 2015 Win64"

```

Comment:
Do we have to drop support for 2013?

Reply:
I can enable it by checking MSVC version seems was due to the introduction of advanced template here https://github.com/dmlc/nnvm/blob/master/include/nnvm/tuple.h#L615